docs/uplink/monitoring-tunnels.md

```diff
@@ -0,0 +1,35 @@
+# Monitoring inlets uplink
+
+Inlets Uplink comes with an integrated Prometheus deployment that automatically collects metrics for each tunnel.
+
+!!! note
+    Prometheus is deployed with Inlets Uplink by default. If you don't need monitoring you can disable it in the `values.yaml` of the Inlets Uplink Helm chart:
+
+    ```yaml
+    prometheus:
+      create: false
+    ```
+
+## Metrics for the control-plane
+
+The control-plane metrics can give you insights into the number of clients that are connected and the number of http requests made to the control-plane endpoint for each tunnel.
+
+| Metric | Type | Description | Labels |
+|--------|------|-------------|--------|
+| controlplane_connected_gauge | gauge | gauge of inlets clients connected to the control plane | `tunnel_name` |
+| controlplane_requests_total | counter | total HTTP requests processed by connecting clients on the control plane | `code`, `tunnel_name`|
+
+## Metrics for the data-plane
+
+The data-plane metrics can give you insights in the services that are exposed through your tunnel.
+
+| Metric | Type | Description | Labels |
+|--------|------|-------------|--------|
+| dataplane_connections_gauge | gauge | gauge of connections established over data plane | `port`, `type`, `tunnel_name` |
+| dataplane_connections_total | counter | total count of connections established over data plane | `port`, `type`, `tunnel_name` |
+| dataplane_requests_total | counter | total HTTP requests processed | `code`, `host`, `method`, `tunnel_name` |
```

Comment:
This is only for HTTP tunnels or?

Reply:
The connections metrics are available for both HTTP and TCP tunnels. The requests metrics are only available for HTTP tunnels. 

This is explained in the text below the table.